README.md

```diff
@@ -0,0 +1,4 @@
+# Python-API-request-script
+This script asks for the client id and secret, and generates a token if the input is valid. It then sends a get request to the Beaver Bus API and prints the data from the request to the user
```

Comment:
Since the script no longer requires user input, you should update this section to explain how to use an example config file